content/zh/_index.md

```diff
@@ -22,7 +22,7 @@ developer_note:
 
 <div class="l-primary-buttons mt-5">
 
-- [了解更多](/docs/what-is-opentelemetry/)
+- [了解更多](/zh/docs/what-is-opentelemetry/)
```

Comment:
@chalin is there a way to express "go to the language specific docs and if you do not find them go to the english ones"? We will need this more often in the future and eventually it will be hard to track which links are pointing where

Reply:
Yes, updated.

Reply:
Thanks for the first few doc pages @windsonsea.

@svrnm - I still need to experiment with Hugo so see what the best way forward is. We might have a few alternatives with tradeoffs to consider. I'll report back here as soon as I can.

Reply:
@svrnm - with the config I've setup, the proper thing to do in the `zh` pages is to lead all URLs with `/zh/` _unless_ there is a wish to intentionally target an `en` page. Does that make sense to you all?

The fallback is implemented via Hugo mounts.